web/Areas/Directory/Views/Index.cshtml

```diff
@@ -0,0 +1,151 @@
+            {
+                   @: <a :href="'https://viper.vetmed.ucdavis.edu/students/sis/default.cfm?Page=AssignPidmAndRedirect&Pidm=' + props.row.spridenId + '&JumpPage=Home'">{{ props.row.spridenId }}</a>
+            } else {
+                   @: {{ props.row.spridenId }}
+            }
+            </div>
+            <div>PIDM: {{ props.row.pidm }}</div>
+            <div>EmployeeID: {{ props.row.employeeId}}</div>
+            <div>IAMID: {{ props.row.iamId }}</div>
+            <div>MothraID: {{ props.row.mothraId }}</div>
+            <div v-if="props.row.mivId > 0">MIVID: {{ props.row.mivId }}</div>
+        </q-td>
+    </template>
+    <template v-slot:body-cell-detail="props">
+        <q-td :props="props">
+            <div><a :href="'https://viper.vetmed.ucdavis.edu/default.cfm?Page=AAUDCheck&mothraID=' + props.row.mothraId + '&loginID=' + props.row.loginId">AAUD Check</a></div>
+            <div><a :href="'https://viper.vetmed.ucdavis.edu/default.cfm?Page=IDCardCheck&LoginID=' + props.row.loginId">ID Check</a></div>
+            <div><a :href="'https://viper.vetmed.ucdavis.edu/default.cfm?Page=UnitHeads&MailID=' + props.row.mailId">MSO/CAO Lookup</a></div>
+            @if (UserHelper.HasPermission(rapsContext, UserHelper.GetCurrentUser(), "SVMSecure.DirectoryUCPathInfo"))
+            {
+                @: <div v-if="props.row.employeeId != ''"><a :href="'https://viper.vetmed.ucdavis.edu/default.cfm?Page=UCPathInfo&emplid=' + props.row.employeeId">UCPath Info</a></div>
+            }
+            @if (UserHelper.HasPermission(rapsContext, UserHelper.GetCurrentUser(), "SVMSecure.CATS.ServiceDesk"))
+            {
+                @: <div v-if="props.row.currentAffiliate == true"><a :href="'https://viper.vetmed.ucdavis.edu/default.cfm?Page=alternatePhoto&iamID' + props.row.iamId + '&mailID=' + props.row.mailId + '&empName=' + props.row.name">Alt. Photo</a></div>
+            }
+        </q-td>
+    </template>
+    <template v-slot:body-cell-links="props">
+        <q-td :props="props">
+            @if (UserHelper.HasPermission(rapsContext, UserHelper.GetCurrentUser(), "SVMSecure.userinfo"))
+            {
+                @: <q-btn :props="props" size="sm" padding="xs" color="primary" square flat icon="person" :href="'userInfo/' + props.row.mothraId"></q-btn>
+            }
+            @if (UserHelper.HasPermission(rapsContext, UserHelper.GetCurrentUser(), "SVMSecure.SU"))
+            {
+                @: <q-btn :props="props" size="sm" padding="xs" color="primary" square flat icon="face" :href="'/EmulateUser/' + props.row.loginId"></q-btn>
+            }
+        </q-td>
+    </template>
+</q-table>
+@section Scripts {
+    <script src="~/js/qtable.js"></script>
+    <script asp-add-nonce="true">
+        createVueApp({
+            data() {
+                return {
+                    userSearch: getItemFromStorage("searchField") ?? "",
```

Comment:
The key for the storage has to be unique across all of VIPER, so consider prefixing searchField with, e.g. directory or some other prefix.